leshan-lwm2m-bsserver/src/main/java/org/eclipse/leshan/bsserver/ConfigurationChecker.java

```diff
@@ -16,33 +16,11 @@
  *******************************************************************************/
 package org.eclipse.leshan.bsserver;
 
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.security.GeneralSecurityException;
-import java.security.PrivateKey;
-import java.security.PublicKey;
-import java.security.cert.Certificate;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-
-import org.eclipse.leshan.bsserver.BootstrapConfig.OscoreObject;
-import org.eclipse.leshan.bsserver.BootstrapConfig.ServerSecurity;
-import org.eclipse.leshan.core.node.InvalidLwM2mPathException;
-import org.eclipse.leshan.core.node.LwM2mPath;
-import org.eclipse.leshan.core.oscore.InvalidOscoreSettingException;
-import org.eclipse.leshan.core.oscore.OscoreSetting;
-import org.eclipse.leshan.core.oscore.OscoreValidator;
-import org.eclipse.leshan.core.security.util.SecurityUtil;
-import org.eclipse.leshan.core.util.StringUtils;
-
 /**
  * Check a BootstrapConfig is correct. This is a complex process, we need to check if the different objects are in
  * coherence with each other.
  */
-public class ConfigurationChecker {
-
-    private OscoreValidator oscoreValidator = new OscoreValidator();
+public interface ConfigurationChecker {
```

Comment:
Just to let you know I change the `ConfigurationChecker` class as an interface and add `DefaultConfigurationChecker` class.